docs/sphinx/source/modelchain.rst

```diff
@@ -389,6 +419,74 @@ shown below.
 .. ipython:: python
 
     mc.infer_ac_model??
+    pvlib.modelchain._snl_params??
+    pvlib.modelchain._adr_params??
+    pvlib.modelchain._pvwatts_params??
+
+ModelChain for a PVSystem with multiple Arrays
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The PVSystem can represent a PV system with a single array of modules, or
+with multiple arrays (see :ref:`multiarray`). The same models are applied to
+all ``PVSystem.array`` objects, so each ``Array`` must contain the appropriate model
+parameters. For example, if ``ModelChain.dc_model='pvwatts'``, then each 
+``Array.module_parameters`` must contain ``'pdc0'``.
+
+When the PVSystem contains multiple arrays, ``ModelChain.results`` attributes
+are tuples with length equal to the number of Arrays. Each tuple's elements
+are in the same order as in ``PVSystem.arrays``.
+
+.. ipython:: python
+
+    from pvlib.pvsystem import Array
+    location = Location(latitude=32.2, longitude=-110.9)
+    inverter_parameters = {'pdc0': 10000, 'eta_inv_nom': 0.96}
+    module_parameters = {'pdc0': 250, 'gamma_pdc': -0.004}
+    array_one = Array(surface_tilt=20, surface_azimuth=200,
+                      module_parameters=module_parameters,
+                      temperature_model_parameters=temperature_model_parameters,
+                      modules_per_string=10, strings=2)
+    array_two = Array(surface_tilt=20, surface_azimuth=160,
+                      module_parameters=module_parameters,
+                      temperature_model_parameters=temperature_model_parameters,
+                      modules_per_string=10, strings=2)
+    system_two_arrays = PVSystem(arrays=[array_one, array_two],
+                                 inverter_parameters={'pdc0': 8000})
+    mc = ModelChain(system_two_arrays, location, aoi_model='no_loss',
+                    spectral_model='no_loss')
+
+    mc.run_model(weather)
+
+    mc.results.dc
+    mc.results.dc[0]
+
+When ``weather`` is a single DataFrame, these data are broadcast and used
+for all arrays. To specify data separately for each array, provide a tuple
+for ``weather`` where each element is a DataFrame containing the required data.
```

Comment:
comment on weather order must be the same as array order or is that too much here?

Reply:
> this suggestion might have been buried/overlooked.

The text is the same as in the suggestion block, am I missing something?

Reply:
Not sure why that shows up here. Maybe I did something weird or maybe there is a github bug. Either way, go to line 417 on the files changed tab: https://github.com/pvlib/pvlib-python/pull/1223/files#diff-d8efb4ce3ee848ab1c98f74e88029b4b8ebc2280178abd49cd9895199297c20eR417

Reply:
Got it. "inferrence" -> "inference"